lib/DDG/Spice/Cargo.pm

```diff
@@ -0,0 +1,27 @@
+package DDG::Spice::Cargo;
+# ABSTRACT: Returns package information from the cargo package manager's registry.
+
+use strict;
+use DDG::Spice;
+
+primary_example_queries "cargo gcc";
+description "Shows a Cargo package";
+name "Cargo";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Cargo.pm";
+icon_url "/i/crates.io.ico";
+topics "sysadmin", "programming";
+category "programming";
+attribution github  => ['https://github.com/tombebbington', 'Tom Bebbington'],
+            twitter => ['https://twitter.com/tomdbebbington', '@Tom Bebbington'];
+
+triggers startend => 'cargo';
```

Comment:
I think we may want to restrict this a little more with "cargo packages" and maybe "rust packages"? as the triggers.
I'm afraid that this will really over-trigger if we assume every query that starts/ends is a valid query for this IA